Installer/package.txt

```diff
@@ -507,26 +507,41 @@ component Runtime.iOS
-		executable ios:iphoneos8.2/Release/dbmysql.lcext as DbMysql base ios:iphonesimulator5.1/Release/dbmysql.dylib
-		file ios:iphoneos8.2/Release/mobile-device-template.plist as "Settings.plist"
-		file ios:iphoneos8.2/Release/mobile-remote-notification-template.plist as "RemoteNotificationSettings.plist"
-		file ios:iphoneos8.2/Release/mobile-url-scheme-template.plist as "URLSchemeSettings.plist"
-		file ios:iphoneos8.2/Release/mobile-splashscreen-template.plist as "SplashscreenSettings.plist"
-		file ios:iphoneos8.2/Release/template-entitlements.xcent as "Entitlements.xcent"
-		file ios:iphoneos8.2/Release/template-store-entitlements.xcent as "StoreEntitlements.xcent"
-		file ios:iphoneos8.2/Release/template-remote-notification-entitlements.xcent as "RemoteNotificationEntitlements.xcent"
-		file ios:iphoneos8.2/Release/template-remote-notification-store-entitlements.xcent as "RemoteNotificationStoreEntitlements.xcent"
-		file ios:iphoneos8.2/Release/template-ResourceRules.plist as "ResourceRules.plist"
-		file ios:iphoneos8.2/Release/Default-568h@2x.png as "Default4InchSplash.png" base ios:Default-568h@2x.png
-		file ios:iphoneos8.2/Release/fontmap as "fontmap"
+	into "[[ToolsFolder]]/Runtime/iOS/Simulator-8_3" place
+		executable ios:iphonesimulator8.3/Release/standalone-mobile[[EditionTagLower]].app/standalone-mobile[[EditionTagLower]] as Standalone base ios:iphonesimulator5.1/Release/standalone-mobile[[EditionTagLower]].app/standalone-mobile[[EditionTagLower]]
+		executable ios:iphonesimulator8.3/Release/revsecurity.dylib as RevSecurity base ios:iphonesimulator5.1/Release/revsecurity.dylib
+		executable ios:iphonesimulator8.3/Release/revpdfprinter.dylib as RevPdfPrinter base ios:iphonesimulator5.1/Release/revpdfprinter.dylib
+		executable ios:iphonesimulator8.3/Release/revzip.dylib as RevZip base ios:iphonesimulator5.1/Release/revzip.dylib
+		executable ios:iphonesimulator8.3/Release/revxml.dylib as RevXml base ios:iphonesimulator5.1/Release/revxml.dylib
+		executable ios:iphonesimulator8.3/Release/revdb.dylib as RevDb base ios:iphonesimulator5.1/Release/revdb.dylib
+		executable ios:iphonesimulator8.3/Release/dbsqlite.dylib as DbSqlite base ios:iphonesimulator5.1/Release/dbsqlite.dylib
+		executable ios:iphonesimulator8.3/Release/dbmysql.dylib as DbMysql base ios:iphonesimulator5.1/Release/dbmysql.dylib
+		file ios:iphonesimulator8.3/Release/mobile-template.plist as "Settings.plist"
+		file ios:iphonesimulator8.3/Release/mobile-remote-notification-template.plist as "RemoteNotificationSettings.plist"
+		file ios:iphonesimulator8.3/Release/mobile-url-scheme-template.plist as "URLSchemeSettings.plist"
+		file ios:iphonesimulator8.3/Release/mobile-splashscreen-template.plist as "SplashscreenSettings.plist"
+		file ios:iphonesimulator8.3/Release/Default-568h@2x.png as "Default4InchSplash.png" base ios:Default-568h@2x.png
+		file ios:iphonesimulator8.3/Release/fontmap as "fontmap"
+	into "[[ToolsFolder]]/Runtime/iOS/Device-8.3" place
```

Comment:
The 'base' for the iphoneos engines (device) should be the previous iphoneos engine version, or nothing if they are no previous versions. As iphoneos will be ARM and iphomesimulator will be x86, there's little point in diffing as they will be completely different.